apps/web/tests/redirects/index.test.ts

```diff
@@ -0,0 +1,138 @@
+import { Link } from "@prisma/client";
+import { describe, expect, test } from "vitest";
+import { IntegrationHarness } from "../utils/integration";
+
+const poweredBy = "Dub.co - Link management for modern marketing teams";
+const domain = "dub.sh";
```

Comment:
this should be the preview URL domain right?

Reply:
You're right. Updated to use the preview URL.